lib/api_client/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 mod client;
+pub mod gql;
```

Comment:
Individual structs/types generated by `graphql_client` are still accessible publically, in cases where types need to be referenced in function sigs, or asserted against. Generally though, this won't be necessary; the traits implement a cleaner abstraction where those implementation details are hidden away.